commons/src/types/Events.ts

```diff
@@ -24,14 +24,14 @@ interface _Event {
 export interface Timespan {
   start_time: number;
   end_time: number;
-  object: "timespan";
+  object?: "timespan";
```

Comment:
Phil to make sure that we can't just pass `object: timespan` when creating event